habit/package.json

```diff
@@ -10,8 +10,12 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@reduxjs/toolkit": "^1.4.0",
```

Comment:
What's this used for?

Reply:
I'm not sure. From where I was reading it was written to use it for redux.

Reply:
Are we using it? @ayu-ayushisood 
If not, let's import only when we'll start using it